doc/gitian-building.md

```diff
@@ -74,11 +74,11 @@ In the VirtualBox GUI click "Create" and choose the following parameters in the
 - File location and size: at least 40GB; as low as 20GB *may* be possible, but better to err on the safe side 
 - Click `Create`
 
-Get the [Debian 8.1 net installer](http://cdimage.debian.org/debian-cd/8.1.0/amd64/iso-cd/debian-8.1.0-amd64-netinst.iso) (a more recent minor version should also work, see also [Debian Network installation](https://www.debian.org/CD/netinst/)).
+Get the [Debian 8.1 net installer](http://cdimage.debian.org/debian-cd/8.2.0/amd64/iso-cd/debian-8.2.0-amd64-netinst.iso) (a more recent minor version should also work, see also [Debian Network installation](https://www.debian.org/CD/netinst/)).
```

Comment:
Agreed,  however https://www.debian.org/CD/ seems to stipulate that you cannot download these over `HTTPS`.